react-icon-pack/package.json

```diff
@@ -5,24 +5,29 @@
   "scripts": {
     "build": "tsup src/index.ts --format cjs,esm --dts",
     "lint": "tsc && eslint src/**",
-    "ci": "yarn lint && yarn build",
-    "prepublishOnly": "yarn ci"
+    "ci": "yarn generateIcons && yarn lint && yarn build",
+    "prepublishOnly": "yarn ci",
+    "generateIcons": "npx @svgr/cli --typescript --icon --replace-attr-values '#000'='currentColor' --out-dir dist/icons -- ../icons"
   },
   "peerDependencies": {
     "react": "^18.2.0",
     "react-dom": "^18.2.0"
   },
   "devDependencies": {
+    "@svgr/cli": "^8.1.0",
+    "@types/lodash": "^4.17.0",
+    "@types/node": "^20.12.7",
     "@types/react": "^18.2.15",
     "@types/react-dom": "^18.2.7",
     "@typescript-eslint/eslint-plugin": "^6.11.0",
     "@typescript-eslint/parser": "^6.11.0",
     "eslint": "^8.53.0",
     "eslint-plugin-react": "^7.33.2",
     "eslint-plugin-react-hooks": "^4.6.0",
+    "lodash": "^4.17.21",
```

Comment:
I would avoid installing full lodash lib as an icon lib dependency, seems redundant to me, as it's quite huge one. Maybe we can just install only what we need, like `lodash/camelCase` and `lodash/startCase` or at least this functions should be imported like `import sth from 'loadash/something'`, this will result in smaller bundle size